.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  env: {
+    node: true,
+    amd: true,
+    es6: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
+    sourceType: 'module', // Allows for the use of imports,
+  },
+  ignorePatterns: ['dist/'],
+  overrides: [
+    {
+      files: ['**/*.ts', '**.*.tsx'],
```

Comment:
why `.tsx`? this module will not work with JSX

Reply:
Is a copy pasta of an existing setup

Reply:
ok, you can remove it. We don't have react in this project

Reply:
I will disagree, it will not introduce any harmful behavior and is easier to know if the file is in sync with other projects.